includes/documents/class-wcpdf-sequential-number-store.php

```diff
@@ -173,6 +178,22 @@ public function get_last_date( $format = 'Y-m-d H:i:s' ) {
 
 		return $formatted_date;
 	}
+
+	/**
+	 * @return bool
+	 */
+	public static function store_name_exists( $store_name ) {
```

Comment:
see other comment about `$store_exists`

Reply:
Are you referring to the `static`?

Reply:
I'm referring to this comment: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/153/files#r577528349